apps/correct-mistakes/translations/src/en/fields.json

```diff
@@ -47,7 +47,7 @@
   },
   "brp-card": {
     "label": "BRP number",
-    "hint": "For example, ZR1000452"
+    "hint": "For example, RZ1000452"
```

Comment:
is there a reason why this was changed?

Reply:
Yes, apparently BRP numbers always start with an 'R' so it wasn't a good hint!

Reply:
ok good stuff then